### PR TITLE
Performance tweaks

### PR DESCRIPTION
**Pull request by odinuge**
screenshot from 2019-03-08 16-42-31

### CODE REVIEW

File: config/webpack.client.js [hunk 1]
```diff
@@ -101,7 +103,13 @@ module.exports = (env, argv) => {
       })
     ]),
     resolve: {
-      modules: [root, 'node_modules']
+      modules: [root, 'node_modules'],
+      alias: {
+        lodash: 'node_modules/lodash-es',
```
Comment (SmithPeder) [inline config/webpack.client.js]: Does this ensure that every time someone imports something from `lodash` the request gets routed to `loadash-es`? That's very clever 💯

File: package.json [hunk 2]
```diff
@@ -50,10 +50,10 @@
     "js-cookie": "^2.1.4",
     "jsdom": "^11.6.2",
     "jwt-decode": "2.2.0",
-    "lodash": "^4.17.11",
-    "medium-draft": "beta",
+    "lodash-es": "^4.17.11",
+    "medium-draft": "^0.6.0-beta1",
```
Comment (SmithPeder) [inline package.json]: Is this new beta1 `editor` flawless 👍 hehe

File: config/webpack.client.js (hunk 1 above)
Comment (odinuge) [reply config/webpack.client.js]: Yes! And then only the stuff we use is added to the bundle!

File: package.json (hunk 2 above)
Comment (odinuge) [reply package.json]: Hahahah, yeee. I checked for an update, but we were running the latest version :/ (as you can see in yarn.lock)

**Pull request merged**